docs/package.json

```diff
@@ -20,7 +20,7 @@
     "@mdx-js/react": "^1.5.1",
     "@mdx-js/tag": "^0.20.3",
     "@next/mdx": "^9.4.0",
-    "@next/plugin-google-analytics": "^9.5.5",
+    "@next/plugin-google-analytics": "^10.0.3",
```

Comment:
any certain reason this dependency is bumped?

Reply:
I got `Next.js plugin versions must match the Next.js version being used, received @next/plugin-google-analytics@9.5.5 need 10.0.3` error when running `npm run dev`.